### DISCUSSION TITLE
three probability responses must sum to 100

### DISCUSSION THREAD

**Discussion by noa-no-h** [jspsych/jsPsych#3446] [Q&A]
I'm asking for participants' probability estimates for four events and I'd like these to sum to 100. I would like to show the total probability (dynamically calculated) below the four percentages they have entered and highlight it red if it is not 100. I'll keep playing around to see if I can get this to work on my own but I first wanted to check if anyone had already done this or had advice for the obvious plugin to use or direction to go in if I wanted to program this. My first instinct is to use jsPsychSurvey, but please let me know if anyone has any other recommendations.

Thanks!

**Comment by ChristopheBossens** [comment]
Were you able to solve this already? I think the jsPsychSurveyText plugin should work here, combined with some custom javascript code to handle calculation of the responses. Let me know if you need help!
